### PR TITLE
DOC: Updated cookbook to show usage of Grouper instead of TimeGrouper…

### PR DESCRIPTION
**Pull request by manikbhandari**
Added links to show usage of Grouper and removed SO links to TimeGrouper- #16747

 - [ ] closes #xxxx
 - [ ] tests added / passed
 - [ ] passes ``git diff upstream/master --name-only -- '*.py' | flake8 --diff``
 - [ ] whatsnew entry

### CODE REVIEW

File: doc/source/cookbook.rst [hunk 1]
```diff
@@ -776,11 +776,14 @@ Resampling
 
 The :ref:`Resample <timeseries.resampling>` docs.
 
-`TimeGrouping of values grouped across time
-<http://stackoverflow.com/questions/15297053/how-can-i-divide-single-values-of-a-dataframe-by-monthly-averages>`__
```
Comment (jorisvandenbossche) [inline doc/source/cookbook.rst]: I would paybe try to update the stackoverflow post?

In any case, the Grouper docs in itself are not a good reference for "Using Grouper to group values across time"

Comment (manikbhandari) [inline doc/source/cookbook.rst]: @jorisvandenbossche thank you for reviewing. I'm sorry I'm new to this, it'll be great if you could guide me on this.

I'll definitely try and update the stackoverflow post. 

> In any case, the Grouper docs in itself are not a good reference for "Using Grouper to group values across time"

Should I instead include some examples to show the usage of Grouper to group values across time?

File: doc/source/cookbook.rst [hunk 2]
```diff
@@ -776,11 +776,17 @@ Resampling
...
+`Using Grouper instead of TimeGrouper
```
Comment (jreback) [inline doc/source/cookbook.rst]: you can still say ``time grouping`` here.

File: doc/source/cookbook.rst [hunk 3]
```diff
@@ -776,11 +776,17 @@ Resampling
...
+<https://stackoverflow.com/questions/15297053/how-can-i-divide-single-values-of-a-dataframe-by-monthly-averages>`__
 
-`TimeGrouping #2
-<http://stackoverflow.com/questions/14569223/timegrouper-pandas>`__
+`Grouper#2
```
Comment (jreback) [inline doc/source/cookbook.rst]: can you give a more informative title (and again you can say 'time grouping')

File: doc/source/cookbook.rst [hunk 4]
```diff
@@ -776,11 +776,17 @@ Resampling
...
+<https://stackoverflow.com/questions/33637312/pandas-grouper-by-frequency-with-completeness-requirement>`__
+
+`Valid frequency arguments to Grouper
+<http://pandas.pydata.org/pandas-docs/stable/timeseries.html#offset-aliases>`__
+
+`Grouping using multiindex
```
Comment (jreback) [inline doc/source/cookbook.rst]: Grouping using a MultiIndex

**Review by jreback — CHANGES_REQUESTED** [review]

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks!

**Pull request merged**